deploy/example_catalog/cr-app-tensorflowcpu.json

```diff
@@ -54,11 +54,11 @@
             }
         ],
         "defaultConfigPackage": {
-            "packageURL" : "file:///opt/configscripts/appconfig.tgz"
+            "packageURL" : "file:///opt/configscript/appconfig-1.8.tgz"
```

Comment:
(trivial): Since the version numbers do not serve any purpose, we could get rid of them in the next version.

Reply:
agreed